lib/Conversion/TorchToTcp/Misc.cpp

```diff
@@ -82,6 +82,8 @@ class ConvertAtenBroadcastLikeOps : public OpConversionPattern<AtenOpT> {
 
     SmallVector<int64_t> axes;
     SmallVector<Value> resultShape;
+    ArrayRef<int64_t> newInputShape =
+        input.getType().dyn_cast<RankedTensorType>().getShape();
```

Comment:
This seems the same a the variable `inputShape` that already exists?

Reply:
`inputShape` is before the `broadcastRankInLeadingDims`. We will need to use the `newInputShape`, which is after the rank expansion (if applicable) to match the rank of output shape